docs/experience/browser-memory.md

```diff
@@ -13,7 +13,7 @@ Browsers manage the memory used by web pages automatically without user interact
 ## Use Cases
 There are many use cases where having insight into the browser's memory usage can be crucial to identify and fix problems as soon as possible. Those include:
 
- * Identifying places in your application resulting in high memory usage
+ * Identifying application workflows resulting in high memory usage including unbound growth over time
```

Comment:
The unbound growth over time - isn't that the same thing as detecting memory leaks, which is the new/last point?

Reply:
Hmm... you are right, let me move the last point as the first and remove the one we are discussing.